packages/core/src/api/expect.ts

```diff
@@ -18,7 +18,7 @@ import {
   setState,
 } from '@vitest/expect';
 import type { RstestExpect, TestCase, WorkerState } from '../types';
-import { createExpectPoll } from './poll';
+import { createExpectPoll } from './pool';
```

Comment:
why rename it?

Reply:
My bad, I thought it was `pool` of tinypool 😅.